straxen/plugins/daqreader.py

```diff
@@ -259,9 +279,84 @@ def compute(self, chunk_i):
             # Ensure the offset is a whole digitizer sample
             records["time"] += self.dt * (self.t0 // self.dt)
 
-        result = self.chunk(
-            start=self.t0 + break_pre,
-            end=self.t0 + break_post,
-            data=records)
-        print(f"Read chunk {chunk_i:04d}, {result} from DAQ")
+        # Split records by channel
+        result_arrays = split_channel_ranges(
+            records,
+            np.asarray(list(self.config['channel_map'].values())))
+        del records
+
+        # Convert to strax chunks
+        result = dict()
+        for i, subd in enumerate(self.config['channel_map']):
+
+            # Ignore data from the 'blank' channels, corresponding to
+            # channels that have nothing connected
+            if subd.endswith('blank'):
+                continue
+
+            result_name = 'raw_records'
+            if subd != 'tpc':
+                result_name += '_' + subd
+            result[result_name] = self.chunk(
+                start=self.t0 + break_pre,
+                end=self.t0 + break_post,
+                data=result_arrays[i],
+                data_type=result_name)
```

Comment:
Shouldn't this be raw_records?

Reply:
We are looping over the different subdetectors here and creating a chunk for each. So this will take on several values; raw_records is one of them.

Reply:
Ah okay, I thought it wasn't creating the other datakinds correctly. However, I see that the error I ran into is due to the omission of the aqmon_nv here. ttps://github.com/XENONnT/straxen/blob/ff98a67e42d922f2cc62d36cfdcfbf84a7635233/straxen/contexts.py#L35

Reply:
Good catch, I removed the line